target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -418,6 +418,9 @@ ramips_board_detect() {
 	*"R6220")
 		name="r6220"
 		;;
+	*"RAK633")
```

Comment:
Drop this change and use `<vendor>,<boardname>` from the dts instead of `rak633`.